common/achievement/get.ts

```diff
@@ -0,0 +1,71 @@
+import { prisma } from '../prisma';
+import { Achievement_template, User_achievement, achievement_action_type_enum, achievement_type_enum } from '../../graphql/generated';
+import { Step, Achievement, achievement_state } from '../../graphql/types/achievement';
+
+const getUserAchievements = async (userId: string): Promise<Achievement[]> => {
```

Comment:
Why not (user: User)? Unlike a string, this can not be misused, and if we have a userId we have a user anyways ;)

Reply:
https://softwareengineering.stackexchange.com/a/80092 - Prefer code quality over performance unless you can really prove a measurable difference (which I doubt exist in this case)